h/static/scripts/directive/annotation.js

```diff
@@ -254,11 +254,6 @@ function AnnotationController(
     /** Whether or not this annotation is private. */
     vm.isPrivate = false;
 
-    /** Copy isSidebar from $scope onto vm for consistency (we want this
-      * directive's templates to always access variables from vm rather than
-      * directly from scope). */
-    vm.isSidebar = $scope.isSidebar;
```

Comment:
This is no longer needed because of the use of `bindToController` in the annotation directive